### PR TITLE
feat: [CHAOS-1724]: added FF, sidebar navlink and RBAC for Security Governance

### PR DESCRIPTION
**Pull request by SahilKr24**
### Summary

Added RBAC, side nav links and Feature Flag for Security Governance.
Also passed 2 components to child app:
```
 <FreezeWindow />
 <UserGroupInput />
 ```

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by pankajsingh-harness — APPROVED** [review]

**Review by knagurski — APPROVED** [review]
I can't see any issues here 👍🏻

**Review by tan-nhu — APPROVED** [review]

**Review by hrishavjha — APPROVED** [review]

File: src/modules/10-common/RouteDefinitions.ts [hunk 1]
```diff
@@ -2139,6 +2139,10 @@ const routes = {
     ({ orgIdentifier, projectIdentifier }: Partial<ProjectPathProps>) =>
       `/chaos/orgs/${orgIdentifier}/projects/${projectIdentifier}/chaos-dashboards`
   ),
+  toChaosSecurityGovernance: withAccountId(
+    ({ orgIdentifier, projectIdentifier }: Partial<ProjectPathProps>) =>
```
Comment (abhinavrastogi-harness) [inline src/modules/10-common/RouteDefinitions.ts:2143]: why Partial? looks like orgid and projectid are always required

Comment (abhinavrastogi-harness) [reply src/modules/10-common/RouteDefinitions.ts:2143]: is this page available only in project scope? if not, you can use `getScopeBasedRoute` util instead

File: src/modules/75-chaos/RouteDestinations.tsx [hunk 2]
```diff
@@ -246,6 +248,19 @@ export default function ChaosRoutes(): React.ReactElement {
     }
   })
 
+  RbacFactory.registerResourceTypeHandler(ResourceType.CHAOS_SECURITY_GOVERNANCE, {
```
Comment (abhinavrastogi-harness) [inline src/modules/75-chaos/RouteDestinations.tsx:251]: shouldn't this be condition on the FF `CHAOS_SECURITY_GOVERNANCE`?

**Review by abhinavrastogi-harness — APPROVED** [review]

File: src/modules/10-common/RouteDefinitions.ts (hunk 1 above)
Comment (SahilKr24) [reply src/modules/10-common/RouteDefinitions.ts:2143]: This is only available in project scope, will update this in a subsequent PR.

File: src/modules/75-chaos/RouteDestinations.tsx (hunk 2 above)
Comment (SahilKr24) [reply src/modules/75-chaos/RouteDestinations.tsx:251]: In that case, will add other RBAC Resource Type under their own Feature Flags also like Gamedays
Tracking this here [CHAOS-1911](https://harness.atlassian.net/browse/CHAOS-1911)

**Pull request merged**